addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -275,6 +275,17 @@ def _import_fill_invoice_form(self, journal, tree, invoice_form, qty_factor):
                 logs.append(_("Could not retrieve currency: %s. Did you enable the multicurrency option and "
                               "activate the currency ?", currency_code_node.text))
 
+        # ==== Bank Details ====
+
+        bank_detail_nodes = tree.findall('.//{*}SpecifiedTradeSettlementPaymentMeans')
+        bank_details = [(
+            bank_detail_node.findtext('{*}PayeePartyCreditorFinancialAccount/{*}IBANID') or bank_detail_node.findtext('{*}PayeePartyCreditorFinancialAccount/{*}ProprietaryID'),
+            bank_detail_node.findtext('{*}PayeeSpecifiedCreditorFinancialInstitution/{*}BICID/{*}ID')
```

Comment:
I don't see this node being used by Factur-X, how did you come up with this ? :)

Reply:
In Factur-X Technical Documentation
![image](https://github.com/odoo/odoo/assets/125546433/c5012cc3-4b5c-4687-8004-08f637836440)

Reply:
Where does that come from ? It's not contained in my version :thinking: (https://fnfe-mpe.org/factur-x/)

Reply:
I downloaded a ZIP file from that same website (received the ZIP through mail) : https://fnfe-mpe.org/?file=Factur-X-1.0.06-2022-03-01-FINAL-EN.zip&nonce=1d33832dd2ecf6fe8ae8b08f1355d1d3&email=aboo%40odoo.com

Reply:
Not sure what happened, I guess they released a new version but didn't increment the pdf version's number, weird :roll_eyes: